### DISCUSSION TITLE
2.52.0版本是不是有问题

### DISCUSSION THREAD

**Discussion by acccccccb** [DouyinFE/semi-design#2075] [General]
2.52.0版本是不是有问题，更新以后项目无法启动，缺少一个样式文件，似乎和semi-animation库有关，升级到2.52.1版本后问题解决了，但是在官方文档更新日志上又看不到2.52.1的更新内容
![image](https://github.com/DouyinFE/semi-design/assets/18065348/6a1857c7-3cea-4807-8c80-0c7a39613dff)

**Comment by SudoUserReal** [comment]
周日修复的，还没来及更新官网 changelog

**Comment by SudoUserReal** [comment]
现在changelog 应该已经更了
